### DISCUSSION TITLE
Binary permalink

### DISCUSSION THREAD

**Discussion by Aqua1ung** [rustdesk/rustdesk#5870] [Feature Request] [answered]
Would it be possible to provide a download permalink to the latest nightly release binary please? This way those of us who rely on [manual updates from Git ](https://github.com/rustdesk/rustdesk/releases/tag/nightly)should be able to script the update process to some extent, instead of having to guess the file name of the latest nightly.

**Comment by SergeyMy** [comment]
Nightly build happens every night regardless of code changes! what is the point of the link?

**Comment by Aqua1ung** [comment, marked as answer]
This should do it:
```
version=$(curl -s -L -D - https://github.com/rustdesk/rustdesk/releases/expanded_assets/nightly | grep -n -m 1 x86_64.rpm | sed -n 's/^.*desk-//p' | sed -n 's/\.x86.*$//p') # Grab the nightly version number.
wget -N https://github.com/rustdesk/rustdesk/releases/download/nightly/rustdesk-$version.x86_64.rpm # Download Rustdesk nightly.
```
